js/ProgressView.windows.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ * @flow strict-local
+ */
+'use strict';
+
+import {requireNativeComponent, StyleSheet} from 'react-native';
+import React from 'react';
+import type {ImageSource} from 'react-native/Libraries/Image/ImageSource';
+import type {ColorValue} from 'react-native/Libraries/StyleSheet/StyleSheetTypes';
+import type {ViewProps} from 'react-native/Libraries/Components/View/ViewPropTypes';
+
+const RNCProgressView = requireNativeComponent(
+  'RNCProgressViewWindows',
+);
+
+type WindowsNativeProps = $ReadOnly<{|
+  ...ViewProps,
+  progressViewStyle?: ?('default' | 'bar'),
+  progress?: ?number,
+  progressTintColor?: ?ColorValue,
+  trackTintColor?: ?ColorValue,
+  isIndeterminate?: ?Boolean, 
+|}>;
+
+const styles = StyleSheet.create({
+  progressWindows: {
+    height: 5
+  },
+});
+
+export default class ProgressViewWindows extends React.Component<WindowsNativeProps> {
```

Comment:
Also, it should be quite easy to rewrite this a function component, which is the new recommended way for writing React.